network-api/networkapi/buyersguide/templates/fragments/product_criterion.html

```diff
@@ -1,4 +1,4 @@
-<div class="criterion criterion-{{ value|lower }} {{criterion_name}} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
+<div class="criterion value-{{ value|lower}}">
```

Comment:
Should we update selectors in `product.scss` from `.criterion-blah` to `.value-blah` as well?

Reply:
I don't think we even need special rules for these now anymore, since we're not showing "a different face for different values" anymore, so we might not even need `value-...` but I kept them in just in case they would be useful for something.

Reply:
kk i can handle the cleanup work in my styling PR